Model/Sync/Catalog/Processor.php

```diff
@@ -173,7 +173,11 @@ public function process($forceSyncProducts = [], $storeIds = [])
                         $forceSyncProductIds = $forceSyncProducts[$storeId] ?? $forceSyncProducts;
                         $collection = $this->getCollectionForSync($forceSyncProductIds);
                         $this->isImmediateRetry = false;
-                        $this->syncItems($collection->getItems(), $storeId);
+
+                        $wereProductCreationAttemptsSuccessful = $this->syncItems($collection->getItems(), $storeId);
```

Comment:
I suspect this isn't true anymore?
it should return `hasFailedCreatingAnyProduct` and not `wereProductCreationAttemptsSuccessful`.

Reply:
You're very right, fixed.